geoips/plugins/modules/readers/ssmis_binary.py

```diff
@@ -183,13 +193,13 @@ def read_ssmis_data_file(fname, metadata_only=False):
     sensor_scan_angle = 45.0
     satellite_altitude = 859
 
-    bad_value = -999
+    bad_value = -999  # NOQA
```

Comment:
Again, you can probably just comment out the variables that are truly not used

Reply:
Just did that